src/string-formatter.ts

```diff
@@ -0,0 +1,44 @@
+import {format} from "util";
+
+export class StringFormatter {
+  public format(parameters: any[]): string {
+    const [, ...textParams] = parameters;
+    let [textFormat] = parameters;
+    let text;
+
+    try {
+      // Our sanitizer gives better results, try with it first
+      textFormat = this.toString(textFormat);
+    } catch {
+      // Fallback to `format`
+      textFormat = format(textFormat);
+    }
+
+    if (textParams.length > 0) {
+      text = format(textFormat, ...textParams);
+    } else {
+      text = textFormat;
+    }
+
+    return text;
+  }
```

Comment:
That `any` can be removed to an `unknown`. I changed this in the last round of changes of the Web SDK. 

```suggestion
  public format(parameters: unknown[]): string {
    const [, ...textParams] = parameters;
    let [textFormat] = parameters as string[];
    let text: string;

    try {
      // Our sanitizer gives better results, try with it first
      textFormat = this.toString(textFormat);
    } catch {
      // Fallback to `format`
      textFormat = format(textFormat);
    }

    if (textParams.length > 0) {
      text = format(textFormat, ...textParams);
    } else {
      text = textFormat;
    }

    return text;
  }
```